examples/interaction/tools/subcoordinates_zoom.py

```diff
@@ -61,10 +71,31 @@
         tool.level = level
     }
 }
-""")
+"""))
+
+hit_test_switch = Switch(active=hit_test)
+hit_test_switch.js_on_change("active", CustomJS(
+    args=dict(tool=ywheel_zoom),
+    code="""
+export default ({tool}, obj) => {
+    tool.hit_test = obj.active
+}
+"""))
+
+only_hit_switch = Switch(active=only_hit)
+only_hit_switch.js_on_change("active", CustomJS(
```

Comment:
Yes, which reflects the API, i.e. `hit_test_behavior` has no function unless `hit_test` is enabled. Of course it would be better from UI perspective to have `only_hit_switch` enabled only when `hit_test_switch` is active.